Sources/Purchasing/Offerings.swift

```diff
@@ -93,4 +96,48 @@ public extension Offerings {
         return description
     }
 
+    /**
+     Retrieves a current offering for a placement identifier, use this to access offerings defined by targeting
+     placements configured in the RevenueCat dashboard, 
+     e.g. `offerings.getCurrentOffering(forPlacement: "placement_id")`.
+     */
+    @objc(getCurrentOfferingForPlacement:)
+    func getCurrentOffering(forPlacement placementIdentifier: String) -> Offering? {
+        guard let placements = self.placements else {
+            return nil
+        }
+
+        let returnOffering: Offering?
+        if let explicitOfferingId: String? = placements.offeringIdsByPlacement[placementIdentifier] {
+            // Don't use fallback since placement id was explicity set in the dictionary
+            returnOffering = explicitOfferingId.flatMap { self.all[$0] }
+        } else {
+            // Use fallback since the placement didn't exist
+            returnOffering =  placements.fallbackOfferingId.flatMap { self.all[$0]}
+        }
```

Comment:
`placements.offeringIdsByPlacement[placementIdentifier]` returns at `Offering??` because `placements.offeringIdsByPlacement` is a `[String: String?]`

The result of could be an explicit `nil` value. In that case, we don't want to use the fallback offering.

If `placements.offeringIdsByPlacement[placementIdentifier]` is not found and goes into the `else`, it means no key was found and we should use the fallback offering.